third_party/rust/chromium_crates_io/BUILD.gn.hbs.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/third_party/rust/chromium_crates_io/BUILD.gn.hbs b/third_party/rust/chromium_crates_io/BUILD.gn.hbs
+index 4cf2f4f5f8b..aa598c2dcf1 100644
+--- a/third_party/rust/chromium_crates_io/BUILD.gn.hbs
++++ b/third_party/rust/chromium_crates_io/BUILD.gn.hbs
+@@ -1,6 +1,7 @@
+-# Copyright 2023 The Chromium Authors
+-# Use of this source code is governed by a BSD-style license that can be
+-# found in the LICENSE file.
++# Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
We probably need to keep the original copyright here?

Reply:
Done